inc/sai.h

```diff
@@ -140,6 +142,8 @@ typedef enum _sai_api_t
     SAI_API_MY_MAC           = 45, /**< sai_my_mac_api_t */
     SAI_API_IPSEC            = 46, /**< sai_ipsec_api_t */
     SAI_API_GENERIC_PROGRAMMABLE = 47, /**<sai_generic_programmable_t */
+    SAI_API_ARS              = 48, /**< sai_ars_api_t */
```

Comment:
Was there a design document in the earlier version of this PR? If yes, can you please add it in this PR too.

Reply:
https://github.com/opencomputeproject/SAI/pull/1658/commits/05e9558fe4d467f3b0881d6e6ca3e57b0bd923ba

Reply:
Design Doc has been added with this commit